pages/03.Howtos/14.add-ptr-records/docs.en.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'How to add PTR records'
+date: '17-09-2020 18:00'
+taxonomy:
+    category:
+        - docs
+---
+
+## How to add PTR records for an existing floating IP
+
+### Overview
+
+This Document will show you the essential steps to add a PTR record for an existing floating IP.
+
+For a complete overview see the [networking reference guide](../../04.Reference/08.network/docs.en.md) and the [dns reference guide](../../04.Reference/07.dns/docs.en.md).
+
+### Prerequisites
+
+* You know the basics of using the [OpenStack CLI-Tools](../../03.Howtos/02.openstack-cli/docs.en.md).
+* You know the basics of the Designate DNS service as shown in the [DNSaaS tutorial](../../02.Tutorials/09.dnsaas/docs.en.md).
+
+You may need to install the designate client (plugin).
+
+```shell
+(sudo) pip install python-openstackclient python-designateclient
+```
+
+### Create your zone
+
+We only allow PTR records for zones that you own. You can use a zone that you already created before this how-to.
+
+If you want to create a subzone for the purpose of delegating it to a different project, you may want to work through [this howto](../../02.Tutorials/09.dnsaas/docs.en.md) first.
+
+If you want to practice with a test domain, you can create an empty zone like this:
+
+```shell
+$ openstack zone create --email email@domain.example ptrhowto.example.
++----------------+--------------------------------------+
+| Field          | Value                                |
++----------------+--------------------------------------+
+| action         | CREATE                               |
+| attributes     |                                      |
+| created_at     | 2019-06-28T15:27:00.000000           |
+| description    | None                                 |
+| email          | email@domain.example                 |
+| id             | 01234567-89ab-cdef-0123-456789abcdef |
+| masters        |                                      |
+| name           | ptrhowto.example.                    |
+| pool_id        | 14234f0f-1234-4444-6789-758006f43802 |
+| project_id     | 0123456789abcdef0123456789abcdef     |
+| serial         | 1561735620                           |
+| status         | PENDING                              |
+| transferred_at | None                                 |
+| ttl            | 21600                                |
+| type           | PRIMARY                              |
+| updated_at     | None                                 |
+| version        | 1                                    |
++----------------+--------------------------------------+
+```
+
+### Create a network and a server
+
+You need at least one network and one server. For testing purposes, you can create a network and a server by following the [first steps tutorial](../../02.Tutorials/01.firststeps/docs.en.md) or the [single lampserver tutorial](../../02.Tutorials/03.single-lamp-server/docs.en.md).
```

Comment:
Maybe reference the other tutorials as ...by following the tutorials ["First Steps"](...) or ["Single LAMP Server"](...)

Reply:
```suggestion
You need at least one network and one server. For testing purposes, you can create a network and a server by following the tutorials "[first steps](../../02.Tutorials/01.firststeps/docs.en.md)" or "[single LAMP server](../../02.Tutorials/03.single-lamp-server/docs.en.md)".
```